umap/views.py

```diff
@@ -114,7 +115,7 @@ def get_context_data(self, **kwargs):
         manager = Map.objects if owner else Map.public
         maps = manager.filter(Q(owner=self.object) | Q(editors=self.object))
         maps = maps.distinct().order_by('-modified_at')[:50]
-        per_page = 10 if owner else self.per_page
+        per_page = getattr(settings, 'UMAP_MAPS_PER_PAGE_OWNER', 10) if owner else getattr(settings, 'UMAP_MAPS_PER_PAGE', self.per_page)
```

Comment:
This line too needs to be under 80 ;)
Actually, here is something to make the code a bit simpler: also add all the settings that uMap expects in settings/base.py, so we are sure they exist, and so you can do `settings.XXXX` instead of `getattr(settings, 'XXX')`.
Makes sense? :)